src/coreclr/vm/vars.hpp

```diff
@@ -99,6 +99,8 @@ struct LOADERHANDLE__
 };
```

Comment:
Nit: You can delete `struct LOADERHANDLE__`. Looks like left-over from earlier definition of `LOADERHANDLE`.